### PR TITLE
Pepper 439 441 os2 germline pdfs v3

### PR DESCRIPTION
**Pull request by ssettipalli**
## Context

Pepper-439  Pepper-441 Osteo germline consent addendum pdfs v3
pdfs generated from activities: GERMLINE_CONSENT_ADDENDUM and GERMLINE_CONSENT_ADDENDUM_PEDIATRIC

PR has some additional typo and text fixes .

## FUD Score

_Overall, how are you feeling about these changes?_

- [ x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ x] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [x ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/osteo/i18n/en.conf [hunk 1]
```diff
@@ -567,7 +567,7 @@
...
     "s4_info2": "If you agree to join this study, your parent will be asked to share information about where you were treated and you will be asked to provide a saliva sample with a kit sent by the project team. You can also choose to share blood sample(s), if you want. After that, the project team will take care of the rest. They will request medical records and get cancer samples and process the blood and saliva kits. All of this information will help researchers better understand cancers.",
-    "s4_info3": "If you choose to share blood samples(s), there are small risks from getting a sample of blood. You may have pain and swelling at the site of the blood draw.",
+    "s4_info3": "If you choose to share blood samples, there are small risks from getting a sample of blood. You may have pain and swelling at the site of the blood draw.",
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/i18n/en.conf:570]: typo fix

File: pepper-apis/studybuilder-cli/studies/osteo/patches/consent-version-2.conf [hunk 2]
```diff
@@ -7,7 +7,7 @@
    "translation-updates":[
         {
             "varName" : "osteo_consent_assent_s4_child_intro_detail_p3",
-            "newValue" : "If you choose to share blood samples(s), there are small risks from getting a sample of blood. You may have pain and swelling at the site of the blood draw. <br/> <br/> Your parent may also be asked if you both want to be interviewed about what it’s like to be part of this research study. You’ll be able to tell your parent if you do or do not want to give an interview."
+            "newValue" : "If you choose to share blood samples, there are small risks from getting a sample of blood. You may have pain and swelling at the site of the blood draw. <br/> <br/> Your parent may also be asked if you both want to be interviewed about what it’s like to be part of this research study. You’ll be able to tell your parent if you do or do not want to give an interview."
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/consent-version-2.conf:10]: typo fix

File: pepper-apis/studybuilder-cli/studies/osteo/patches/germline-consent-addendum-ped-v3.conf [hunk 3]
```diff
@@ -53,7 +53,7 @@
                 "translations": [
                   {
                     "language": "en",
-                    "text": """If you have questions about the study, please contact the research doctor or study staff listed below by emailing  <a href="mailto:info@lmsproject.org" class="Link">info@lmsproject.org</a> or calling <a href="tel:651-403-5556" class="Link">651-403-5556</a>:"""
+                    "text": """If you have questions about the study, please contact the research doctor or study staff listed below by emailing <a href="mailto:info@osproject.org" class="Link">info@osproject.org</a> or calling <a href="tel:651-602-2020" class="Link">651-602-2020</a>:"""
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/germline-consent-addendum-ped-v3.conf:56]: text fix

**Comment by ssettipalli** [comment]
Deployed to Dev and tested. Looked ok to me.
Attached dev generated pdfs to tickets since I couldn't attach the pdfs here

**Review by otchet-broad — APPROVED** [review]
Thanks Sampath!  LGTM!

**Pull request merged**